edge-functions/json-response/pages/index.tsx

```diff
@@ -0,0 +1,3 @@
+const Page = () => <div>lol</div>
+
+export default Page
```

Comment:
@dominiksipowicz Let's remove this, the example should show a JSON in the index page just as it did before.